src/SFA.DAS.EmployerFinance.Jobs/NServiceBusStartup.cs

```diff
@@ -33,14 +33,15 @@ public class NServiceBusStartup : IStartup
         
         public async Task StartAsync()
         {
-            var endpointConfiguration = new EndpointConfiguration("SFA.DAS.ProviderRelationships.JobsV2")
-                .UseAzureServiceBusTransport(() => _employerFinanceConfiguration.ServiceBusConnectionString, _environmentService.IsCurrent(DasEnv.LOCAL))
+            var endpointConfiguration = new EndpointConfiguration("SFA.DAS.EmployerFinance.JobsV2")
+                .UseAzureServiceBusTransport(() => 
+                    _employerFinanceConfiguration.ServiceBusConnectionString, 
+                    _environmentService.IsCurrent(DasEnv.LOCAL))
                 .UseLicense(_employerFinanceConfiguration.NServiceBusLicense)
                 .UseMessageConventions()
-                .UseSqlServerPersistence(() => _container.GetInstance<DbConnection>())
                 .UseNewtonsoftJsonSerializer()
                 .UseNLogFactory()
-                .UseSendOnly()
```

Comment:
The Jobs endpoint should be send only, could this be added back in?

Reply:
I originally removed it because it will not create the learning transport folders if it is set to send only but i'll add it back in now given we are removing the dummy examples